operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 * [FEATURE] Add support for Vault Agent. When enabled, the Pod annotations for TLS configurable components are updated to allow a running Vault Agent to fetch secrets from Vault and to inject them into a Pod. The annotations are updated for the following components: `admin-api`, `alertmanager`, `compactor`, `distributor`, `gateway`, `ingester`, `overrides-exporter`, `querier`, `query-frontend`, `query-scheduler`, `ruler`, `store-gateway`. #4660
 * [FEATURE] Add documentation to use external Redis support for chunks-cache, metadata-cache and results-cache. #4348
 * [FEATURE] Allow for deploying mixin dashboards as part of the helm chart. #4618
+* [ENHANCEMENT] Update the `rollout-operator` subchart to `0.4.2`. #4780
```

Comment:
There are three entries for rollout-operator updates to `0.4.0`, `0.4.1`, and `0.4.2`. For ease of understanding I think it would be clearer to delete the other entries and consolidate to a single entry that says:
```
* [ENHANCEMENT] Update the `rollout-operator` subchart to `0.4.2`. #4524 #4659 #4780
```